src/orchestrator/src/lib.rs

```diff
@@ -293,6 +293,8 @@ pub struct ServiceConfig<'a> {
     pub disk: bool,
     /// The maximum amount of scratch disk space that the service is allowed to consume.
     pub disk_limit: Option<DiskLimit>,
+    /// Node selector for this service.
+    pub service_node_selector: BTreeMap<String, String>,
```

Comment:
```suggestion
    pub node_selector: BTreeMap<String, String>,
```

(The `service_` is implied by this being part of the `ServiceConfig`; it's needed in the overall config because that's for the orchestrator as a whole, and not just the service.)